config/core/rootfs-images.yaml

```diff
@@ -90,3 +90,20 @@ file_systems:
     ramdisk: bullseye/20221125.0/{arch}/rootfs.cpio.gz
     root_type: ramdisk
     type: debian
+  debian_sid-kselftest_nfs:
+    boot_protocol: tftp
+    nfs: sid-kselftest/20221125.0/{arch}/full.rootfs.tar.xz
```

Comment:
It looks like we may want to check if there is a glitch in a script updating rootfs images URLs.

Reply:
This was added by hand, so it's not an issue with the automated update.

Reply:
@mgalka @gctucker so does the date subdir need to be updated in this PR?  or will it be updated correctly once it's merged?  I'm not sure how the scripts work.

Reply:
It needs to be added first by hand, then the script will update it automatically.  The way it works is it loads the config, iterates over all the entries and tries the new URLs with today's date.  Those that work get updated, otherwise they stay with the previous one.  Then it serialises the config back to YAML again into this file.

Reply:
I can't see `sid-kselftest` rootfs image definition in `rootfs-configs.yaml`, however I can see a PR which adds it and there is a comment https://github.com/kernelci/kernelci-core/pull/1510#issuecomment-1323409325 about missing packages.

Reply:
On the grounds that `sid` is going to be broken a lot of the time, maybe we can just lower the bar to say if it builds then merge it?  That way we'll have the entry with a URL in `rootfs-configs.yaml` which will get automatically updated in each prod update.  Meanwhile, the list of packages can be tweaked as a follow-up etc.